### PR TITLE
Three small changes: play/pause tooltip, mat icon typo, aria recommendation

### PR DESCRIPTION
**Pull request by joanise**

### CODE REVIEW

**Comment by joanise** [comment]
Note: I'm not sure my Spanish translation is OK, with "Déle play/pause", but I was adapting from the existing text. I'm hoping for a native speaker review before merging.

**Review by dhdaines — APPROVED** [review]
I will defer to a native speaker for the Spanish translation (which I find a bit shocking! where's the equivalent of the OQLF?) but aside from that this seems quite sensible.

**Pull request merged**
